src/legacy/ui/public/vis/map/service_settings.js

```diff
@@ -47,7 +47,8 @@ uiModules
         this._showZoomMessage = true;
         this._emsClient = new EMSClient({
           language: i18n.getLocale(),
-          kbnVersion: kbnVersion,
+          appVersion: kbnVersion,
+          appName: 'kibana',
```

Comment:
@thomasneirynck It does not appear we have a constants file for legacy apps? It's okay to hard-code `kibana` here?

Reply:
+1